tests/local/missing-features/issue540.wast

```diff
@@ -0,0 +1,33 @@
+(module binary
+  "\00asm\01\00\00\00"    ;; the header
+
+  "\05\06\01"       ;; a memory section with 1 entry
+  "\01\70\e0\ab\02" ;; definition of the 0th memory
+
+  "\0b\08\01"       ;; a data section with 1 entry
+  "\02\00"          ;; data section entry referencing 0th memory using flag 02 encoding.
+  "\41\ca\0f\0b\00" ;; offset and data for the entry
+)
+
+(module binary
+  "\00asm\01\00\00\00"    ;; the header
+
+  "\05\06\01"       ;; a memory section with 1 entry
+  "\01\70\e0\ab\02" ;; a definition of the 0th memory
+
+  "\0b\08\01"       ;; a data section with 1 entry
+  "\80\00"          ;; data section entry referencing 0th memory using a WebAssembly 1.0 encoding
+                    ;; this encoding is not supported in current WebAssembly draft.
+  "\41\ca\0f\0b\00" ;; offset and data for the entry
+)
```

Comment:
Feel free to make this `assert_malformed/invalid` as appropriate when the flag decoding is replaced with `read_u8`.